geomesa-kafka/geomesa-kafka-datastore/src/test/scala/org/locationtech/geomesa/kafka/KafkaDataStoreTest.scala

```diff
@@ -93,6 +104,17 @@ class KafkaDataStoreTest extends Specification with Logging {
       fw.write()
       Thread.sleep(2000)
 
+      //Check LoadingCache expiration
+      {
+        import cachedConsumerFS._
+        val cachedFeatures = cachedConsumerFS.features.asInstanceOf[Cache[String, FeatureHolder]]
+        cachedFeatures.cleanUp() //remove old entries...hopefully there aren't any
+        cachedFeatures.size() must be equalTo 1
+        Thread.sleep(1100)
```

Comment:
Might be better to break this out into a separate test method and turn off the expiry for the normal test - otherwise you might get some unexpected expirations if someone tries to update the test.